bluesky/callbacks/core.py

```diff
@@ -534,13 +534,17 @@ def start(self, doc):
         self.im = im
         self.ax.set_title('scan {uid} [{sid}]'.format(sid=doc['scan_id'],
                                                       uid=doc['uid'][:6]))
+        self.snaking = doc.get('snaking', (False, False))
```

Comment:
...Or, you know, a test :imp: ... but I would accept a link to the relevant line in plans.py where this happens.